frontend/scripts/react-components/shared/dropdown/dropdown.component.jsx

```diff
@@ -30,104 +60,161 @@ class Dropdown extends React.Component {
-    const { arrowType, selectedValueOverride } = this.props;
+    return optionsToShow.map((item, i) => renderItem(item, i, highlightedIndex, getItemProps));
+  }
 
+  /* eslint-disable react/prop-types */
+  function renderButton({ ref, inputValue, getToggleButtonProps }) {
+    const { arrowType, selectedValueOverride } = props;
     return (
       <button
-        {...getToggleButtonProps()}
-        ref={ref}
-        className={cx('dropdown-selected-item', { [`-${arrowType}`]: arrowType })}
-        onClick={toggleMenu}
+        {...getToggleButtonProps({
+          ref,
+          className: cx('dropdown-selected-item', { [`-${arrowType}`]: arrowType })
+        })}
       >
         {selectedValueOverride || inputValue}
       </button>
     );
-  };
-
-  renderBox = ({ ref, style, placement, getItemProps, selectedItem }) => (
-    <ul ref={ref} style={style} data-placement={placement} className="dropdown-menu">
-      {this.getSelectedOptions(selectedItem).map((item, index) =>
-        this.renderItem(item, index, getItemProps)
-      )}
-    </ul>
-  );
+  }
 
-  render() {
-    const {
-      options,
-      value,
-      onChange,
-      itemToString,
-      color,
-      variant,
-      readOnly,
-      placement
-    } = this.props;
+  function renderContent({
+    ref,
+    style,
+    placement,
+    selectedItem,
+    getItemProps,
+    getMenuProps,
+    highlightedIndex,
+    toggleMenu
+  }) {
+    const decoratedChildren =
+      typeof props.children !== 'undefined' ? (
+        <DropdownContext.Provider
+          value={{ selectedItem, getItemProps, highlightedIndex, toggleMenu }}
+        >
+          {props.children}
+        </DropdownContext.Provider>
+      ) : (
+        undefined
+      );
+    const styleToApply = listHeight ? { ...style, height: listHeight } : style;
     return (
-      <Downshift defaultSelectedItem={value} itemToString={itemToString} onChange={onChange}>
-        {({ getItemProps, isOpen, toggleMenu, getToggleButtonProps, selectedItem, inputValue }) => (
-          <div
-            className={cx('c-dropdown-component', {
-              '-open': isOpen,
-              [`v-${variant}`]: variant,
-              [`color-${color}`]: color,
-              '-read-only': readOnly
-            })}
-          >
-            <Manager>
-              <Reference>
-                {props =>
-                  this.renderButton({ ...props, toggleMenu, getToggleButtonProps, inputValue })
-                }
-              </Reference>
-              {isOpen && options.length > 0 ? (
-                <Popper placement={placement || 'bottom-end'}>
-                  {props => this.renderBox({ ...props, getItemProps, selectedItem })}
-                </Popper>
-              ) : null}
-            </Manager>
-          </div>
-        )}
-      </Downshift>
+      <ul
+        {...getMenuProps({
+          ref,
+          style: styleToApply,
+          'data-placement': placement,
```

Comment:
![image](https://user-images.githubusercontent.com/9701591/54682073-3b6aef80-4b0e-11e9-9ef0-7970f8e62024.png)
I think moving some extra pixels to the top would be needed